lib/scenarios.rb

```diff
@@ -46,28 +48,41 @@ def create_broker_er_in_pending
       end
     end
 
-# Create employee insured
+    # Create employee insured
     def create_employee
       employee_util EMPLOYEE do |employee_util|
         reset_count
-        employee_util.set_employer_values(*::Sample.client_A)
-        write_json employee_util.create_single_employee, employee_util 
+        employee_util.set_employer_values(*::ClientData.client_A)
+        write_json employee_util.create_single_employee, employee_util
       end
     end
 
-# Create individual insured with UQHP
+    # Create individual insured with UQHP
     def create_individual_uqhp
       individual_util INDIVIDUAL_UQHP do |individual_util|
         reset_count
-        write_json individual_util.create_individual_uqhp, individual_util 
+        write_json individual_util.create_individual_uqhp, individual_util
       end
     end
 
-
-# Create individual insured with APTC
+    # Create individual insured with APTC
     def create_individual_aptc
     end
```

Comment:
@benjaminrosenbaum Your comments are predominantly domain related. We can do one of 2 things:

1. I can ping you, learn why some of this may have changed (_can't see my last set of changes cause this, but maybe they did_) and then, work on making the changes.
2. You can make these changes as they might be quite straightforward for you.

Let me know.